thrall/test/lib/ElasticSearchTestBase.scala

```diff
@@ -301,16 +314,23 @@ trait ElasticSearchTestBase extends FreeSpec with Matchers with Fixtures with Be
 
       "removing a lease should update the leases last modified time" in {
         val lease = model.MediaLease(id = Some(UUID.randomUUID().toString), leasedBy = None, notes = Some("A test lease"), mediaId = UUID.randomUUID().toString)
+        val timeBeforeEdit = DateTime.now
         val id = UUID.randomUUID().toString
-        val image = createImageForSyndication(id = UUID.randomUUID().toString, true, Some(DateTime.now()), lease = Some(lease))
+        val image = createImageForSyndication(
+          id = UUID.randomUUID().toString,
```

Comment:
is this meant to be the `id` val above?

Reply:
I hadn't touched this, but looking at it, yes I think so! I'll clean it up.